tests/fft_test.py

```diff
@@ -90,20 +94,16 @@ def testFftnErrors(self, inverse):
     func = np.fft.ifftn if inverse else np.fft.fftn
     self.assertRaisesRegex(
         ValueError,
-        "jax.np.fft.{} only supports 1D, 2D, and 3D FFTs over the innermost axes. "
-        "Got axes None with input rank 4.".format(name),
+        "jax.np.fft.{} only supports 1D, 2D, and 3D FFTs".format(name),
```

Comment:
nit: I like having the full error message in tests (within reason). It makes it easy to review that the error message is clear and doesn't have typos (for both readers and authors changing the error message). Up to you though.